frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,20 @@
+export function Footer() {
+    const getFile = async (path) => {
+        try {
+            const response = await fetch(`/api/files/${path}`);
+            const blob = await response.blob();
+            const pdfBlob = new Blob([blob], { type: 'application/pdf' });
+            const url = URL.createObjectURL(pdfBlob);
+            window.open(url, '_blank');
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    return (
+        <div className='footer_container'>
+            <a className='footer_link' onClick={() => { getFile('terms-of-service') }}>Terms Of Service</a>
```

Comment:
Dlaczego nie utworzyliśmy tu zwykłego linku (bez użycia JS), skoro API zwraca plik?
`<a href='/api/files/terms-of-service'>Terms Of Service</a>`

Reply:
nie można było tak tydzień temu?